galaxy_ng/app/constants.py

```diff
@@ -16,3 +16,290 @@ class DeploymentMode(enum.Enum):
+
+    },
+    "galaxy.add_user": {
+        "name": _("Add user"),
+
+        "object_description": None,
+
+        "global_description": _("Add new users to the system."),
+
+        "ui_category": _("Users"),
+
+    },
+    "galaxy.change_user": {
+        "name": _("Change user"),
+
+        "object_description": _("Edit this user."),
+
+        "global_description": _("Edit any existing user in the system."),
+
+        "ui_category": _("Users"),
+
+    },
+    "galaxy.delete_user": {
+        "name": _("Delete user"),
+
+        "object_description": _("Delete this user."),
+
+        "global_description": _("Delete any existing user in the system."),
+
+        "ui_category": _("Users"),
+
+    },
+    "galaxy.view_user": {
+        "name": _("View user"),
+
+        "object_description": _("View this user."),
+
+        "global_description": _("View any user in the system."),
+
+        "ui_category": _("Users"),
+
+    },
+    "galaxy.add_group": {
+        "name": _("Add group"),
+
+        "object_description": None,
+
+        "global_description": _("Create new groups to the system."),
+
+        "ui_category": _("Groups"),
+
+    },
+    "galaxy.change_group": {
+        "name": _("Change group"),
+
+        "object_description": _("Edit this group"),
+
+        "global_description": _("Edit any existing group in the system."),
+
+        "ui_category": _("Groups"),
+
+    },
+    "galaxy.delete_group": {
+        "name": _("Delete group"),
+
+        "object_description": _("Delete this group."),
+
+        "global_description": _("Delete any group in the system."),
+
+        "ui_category": _("Groups"),
+
+    },
+    "galaxy.view_group": {
+        "name": _("View group"),
+
+        "object_description": _("View this group."),
+
+        "global_description": _("View any existing group in the system."),
+
+        "ui_category": _("Groups"),
+
+    },
+        "ansible.change_collectionremote": {
+        "name": _("Change collection remote"),
+
+        "object_description": _("Edit this collection remote."),
+
+        "global_description": _("Edit any collection remote existing in the system."),
+
+        "ui_category": _("Collection Remotes"),
+
+    },
+    "ansible.view_collectionremote": {
+        "name": _("View collection remote"),
+
+        "object_description": _("View this collection remote."),
+
+        "global_description": _("View any collection remote existing in the system."),
+
+        "ui_category": _("Collection Remotes"),
+
+    },
+    "container.change_containernamespace": {
+        "name": _("Change container namespace permissions"),
+
+        "object_description": _("Edit permissions on this namespace."),
+
+        "global_description": _("Edit permissions on any existing namespace."),
+
+        "ui_category": _("Containers"),
+
+    },
+    "container.namespace_change_containerdistribution": {
+        "name": _("Change containers"),
+
+        "object_description": _("Edit this container."),
```

Comment:
All these `container.namespace_` permissions grant users the ability to modify all objects in the container namespace. This (and all the other permissions in this category) should say use "{ACTION} in this container namespace" for the object_description.

Reply:
Do you have a preference for wording - "...this/any container namespace in the system" vs "..any existing container namespace" vs "...any existing container ns in the system" ? Seems like we could pick one for consistency.